airbyte-integrations/connectors/source-bing-ads/unit_tests/integrations/catalog_builder.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+
+from airbyte_cdk.models import ConfiguredAirbyteCatalog, SyncMode
+
+
+class CatalogBuilder:
```

Comment:
Could you please extend the `CatalogBuilder` class from `airbyte_cdk.test.catalog_builder` and override method `with_stream`?